pages/index.tsx

```diff
@@ -7,22 +7,76 @@ import Head from '../components/Layout/Head';
 import styles from '../styles/Home.module.css';
 import Image from 'next/image';
 import humanImage from '../public/Human.png';
+import translateImage from '../public/translate.png';
+import { useState } from 'react';
+
+import Lottie from 'react-lottie';
+import backgroundAnimation from '../lottie/background.json';
+import globeAnimation from '../lottie/globe.json';
+
+const bgAni = {
+  loop: true,
+  autoplay: true,
+  animationData: backgroundAnimation,
+  rendererSettings: {
+    preserveAspectRatio: 'xMidYMid slice',
+  },
+};
+
+const globeAni = {
+  loop: true,
+  autoplay: true,
+  animationData: globeAnimation,
+  rendererSettings: {
+    preserveAspectRatio: 'xMidYMid slice',
+  },
+};
 
 const Home: NextPage<{ locations: Location[] }> = ({ locations }) => {
+  const [showDrop, setShowDrop] = useState(false);
+
   return (
     <div className={styles.mainContainer}>
       <div className={styles.leftWindow}>
+        <div className={styles.translateIcon}>
+          <Image
+            src={translateImage}
+            alt="translate icon"
+            onClick={() => setShowDrop(!showDrop)}
```

Comment:
safer way to do this (plus more descriptive naming)

```
onClick={(0 => setShowDropdown(prev => !prev)}
```
It is safer to use the previous as its state can not be manipulated somewhere else in the render